fulp_modules/features/toys/code/plushies.dm

```diff
@@ -87,3 +87,20 @@
 	icon = 'fulp_modules/features/toys/icons/toys.dmi'
 	icon_state = "beefman"
 	squeak_override = list('sound/effects/meatslap.ogg'=1)
+
+/obj/item/toy/plush/beefplushie/living_beefplushie
+	desc = "It looks oddly alive. You feel like you should pet it."
+	COOLDOWN_DECLARE(beefplushie_cooldown)
+	#define BEEFPLUSHIE_COOLDOWN_TIME (2 MINUTES)
```

Comment:
Put the define above the code and #undef it at the end

Reply:
like at the very top or just above the obj typepath?

Reply:
undef inside or outside the typepath?

Reply:
im too new at this

Reply:
we'll see if i did it right off examples